test/integration/puppetlabs/puppetserver/testutils.clj

```diff
@@ -375,6 +375,15 @@
   ([puppet-conf-file dest-dir]
    (with-puppet-conf-fixture {"puppet.conf" puppet-conf-file} dest-dir)))
 
+(defmacro with-stub-puppet-conf
+  "Provides the functionality of `with-puppet-conf` but uses a default stub
```

Comment:
yeah, it should be I did a couple of versions of this didn't update the doc correctly, thanks.

Reply:
I forgot to update the docs after an iteration. It _is_ similar in its behavior to what a 0-arity branch of `with-puppet-conf` might be. But it can't be called in the same way/places as the that function.